torch_xla/csrc/tensor_methods.cpp

```diff
@@ -86,6 +84,7 @@
 #include "torch_xla/csrc/ops/triu.h"
 #include "torch_xla/csrc/ops/unsqueeze.h"
 #include "torch_xla/csrc/ops/view.h"
+#include "torch_xla/csrc/tensor.h"
```

Comment:
Hmmm, this is OK, considering the alphabetical order, but first line include should be left alone.
This cpp file is essentially a continuation of tensor.cpp.

Reply:
I haven't done this on my own, this is all `clang-format`. Unless I disable it with `// clang-format off`, it will do this.